winml/test/scenario/models/LICENSE.md

```diff
@@ -0,0 +1,6 @@
+# Licenses
+
+| Model       | Source     | License     |
+| ----------- | ---------- | ----------- |
+| Resnet 50 | https://github.com/keras-team/keras | MIT |
+| Inception v1 | https://github.com/onnx/models (TODO double check this) | MIT |
```

Comment:
Can someone confirm where this comes from? Given the name of the model (`onnxzoo_lotus_inception_v1-dq.onnx`) I assume it's from ONNX model zoo, but I couldn't find it on the model zoo (there are Inception v1 models there, but they don't match this file).

Reply:
I think they don't match because we modified the model to be quantized so we could test dequantization. is that the discrepancy?

---
In reply to: [349407360](https://github.com/microsoft/onnxruntime/pull/2457#discussion_r349407360) [](ancestors = 349407360)

Reply:
Yes, that's it